src/robusta/core/sinks/slack/slack_sink_params.py

```diff
@@ -35,19 +33,43 @@ def validate_channel_override(cls, v: str):
                 )
                 raise ValueError(err_msg)
         return v
+    
+    def normalize_placeholder(cls, s):
+        def repl(match):
+            return cls.normalize_key_string(match.group(0))
+        
+        result = s.replace(ANNOTATIONS_PREF, '').replace(LABELS_PREF, '')
+        # Use regular expression to find and replace inside "${}"
```

Comment:
If I understand correctly, this code assume that the parameters are surrounded by `{}`, but this is not the supported format
See the docs [here](https://docs.robusta.dev/master/configuration/sinks/slack.html#dynamic-slack-channels)

Possible values for the `channel_override` are:
* channel_override: "cluster_name"
* channel_override: "annotations.anno.slack"
* channel_override: "arik-$labels.lusha-test"

In general, it's either `cluster_name`, a label or an annotation (without the `$` sign)
Or, a string template, the the parameters with a `$` sign before the parameter

Reply:
For long sentences that includes special characters (like kubernet.io/owner, etc...) it is crucial to use the ${} pattern.
but the old pattern still works, the logic is preserved for simple $lables.test_label.

Would you like me to add to the docs an explanation ?

Reply:
Got it @levtomer66 , thanks
So if I understand correctly, if your annotation or label includes `.` or `-` or `/`, you need to use the new syntax, with the `{}`, right?
I think we need to add it to the docs, otherwise users won't know about it

The location to add it is [here](https://github.com/robusta-dev/robusta/blob/d4a4c3404208da4c323ddd87ab9f6e4ab02f0479/docs/configuration/sinks/slack.rst?plain=1#L34) somewhere 

You can build the docs locally by running:
```
poetry install -E all
./docs_autobuild.sh
```

Reply:
Exactly, I've added to the docs